assets/queries/k8s/saTokenSecretQuery/query.rego

```diff
@@ -0,0 +1,56 @@
+package Cx
+
+CxPolicy [result] {
+    document := input.document[i]
+    metadata := document.metadata
+    
+    validKind := ["Role","ClusterRole"]
+    ruleTaint := ["get","watch","list","*"]
+    resourcesTaint := ["secrets"]
+    
+    kind := document.kind
+    contains(validKind,kind)
+    name := metadata.name
+    
+    bindingExists(name,kind)
+    
+   	resources := document.rules[_].resources; 
+    some resource
+    	contains(resourcesTaint,resources[resource])
+    
+    rules := document.rules[_].verbs;    
+    some rule
+    	contains(ruleTaint,rules[rule])
+    
+    result := {
+                "documentId":        input.document[i].id,
+                "searchKey":         sprintf("metadata.name=%s.rules.verbs", [metadata.name]),
+                "issueType":        "IncorrectValue",
+                "keyExpectedValue":  sprintf("The %s.rules.verbs should not contain the following verbs: [%s]", [metadata.name,rules]),
+                "keyActualValue":    sprintf("The %s.rules.verbs contain the following verbs: [%s]", [metadata.name,rules])
```

Comment:
This is only for visualization but metadata.name is a value, not a key in the resource, meaning that "resource_name".rules.verbs wouldn't be found, because it's not a key, you should have instead: metadata.name=%s.rules.verbs